### PR TITLE
Terms conditions redesign

### LINKED ISSUE #336: Terms + Conditions redesign
Terms of services are ridiculously large lol

image

### PR DESCRIPTION
**Pull request by saidam90**
resolves #336

### CODE REVIEW

File: src/app/terms/layout.tsx
Comment (dcts) [inline src/app/terms/layout.tsx:5]: Can you make this `max-w-screen-md` (instead of lg)?

**Review by dcts — CHANGES_REQUESTED** [review]
Looks like you based this branch off of your `capitalize-first-letter` branch. 

You can either rebase (remove all changes in translation files) or wait until #478 is merged and then update this PR.

**Comment by saidam90** [comment]
oops you're right, I created this branch incorrectly. for some reason I cannot rebase, it still says `Updates were rejected because the tip of your current branch is behind its remote counterpart`, although I integrated the remote changes with `git pull`. Should I close this PR and open a new one?

**Comment by dcts** [comment]
Yes feel free to close this PR and create a new one, or alternatively I suggest trying this:

```
# inside your forked repo: go to main branch
git checkout main

# pull from origin main
git pull origin main

# go to your feature branch
git checkout terms-conditions-redesign

# merge changes from main
git merge main

# update the PR
git push origin terms-conditions-redesign
```

**Comment by dcts** [comment]
replaced by #481

**Pull request closed without merging**
